crates/cargo-acap-sdk/src/commands/run_command.rs

```diff
@@ -26,27 +28,34 @@ impl RunCommand {
             pass: password,
         } = deploy_options;
 
+        let host = format!("{}:22", address);
+
+        let tcp = TcpStream::connect(host).unwrap();
```

Comment:
Since this function returns `anyhow::Result` we should either return an `Err` or `unwrap` and leave a comment explaining why the unwrap will not panic.